config/routes.rb

```diff
@@ -48,7 +48,10 @@
     resource :notify_callback, only: :create, path: "notify-callback"
 
     namespace :v1 do
-      resources :ecf_participants, path: "participants/ecf", only: %i[index]
+      resources :ecf_participants, path: "participants/ecf", only: %i[index] do
+        member { put :withdraw }
+      end
+
```

Comment:
I think we will need to document this in swagger at some point. Possibly not this PR though.

Reply:
I very much doubt it is done, I don't see anything for `api/v1/participants/ecf/{id}/withdraw` in our swagger docs :P 

You added docs for npq, but this is an ecf endpoint.